main.go

```diff
@@ -1,18 +1,22 @@
 package main
 
 import (
-	"github.com/gin-gonic/gin"
 	"github.com/OpenDataTelemetry/timeseries-api/controller"
+	"github.com/gin-gonic/gin"
 )
 
 func main() {
 
 	r := gin.Default() // Create a new gin router instance
-	api := r.Group("/api/timeseries/v0.1/smartcampusmaua/SmartLights")
+	api := r.Group("/api/timeseries/v0.2/smartcampusmaua")
 	{
-		api.GET("", controller.GetSmartLights)
-		api.GET("deviceName/:nodename", controller.GetSmartLightbyNodeName)
-		api.GET("deviceId/:devEUI", controller.GetSmartLightbyDevEUI)
+		api.GET("SmartLights", controller.GetSmartLights)
+		api.GET("SmartLights/deviceName/:nodename", controller.GetSmartLightbyNodeName)
+		api.GET("SmartLights/deviceId/:devEUI", controller.GetSmartLightbyDevEUI)
+
+		api.GET("WaterTankLevel", controller.GetWaterTankLevel)
```

Comment:
Modify to "WaterTankLevels" since it returns an array of WaterTankLevel sensors.

Reply:
Chnaged as requested